charts/sonarqube-dce/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 # SonarQube Chart Changelog
 All changes to this chart will be documented in this file.
 
+## [5.1.3]
+* Refactor Pdb and Ingress to be compatible with static compatibitly test and 1.19 minimum requirement
+* Add some value file for static comptatibily testing
```

Comment:
We can remove this line from changelog since values files have been removed from the chart